manager/state/raft/raft.go

```diff
@@ -1139,13 +1144,13 @@ func (n *Node) send(messages []raftpb.Message) error {
 		}
 
 		n.asyncTasks.Add(1)
-		go n.sendToMember(members, m)
+		go n.sendToMember(ctx, members, m)
 	}
 
 	return nil
 }
 
-func (n *Node) sendToMember(members map[uint64]*membership.Member, m raftpb.Message) {
+func (n *Node) sendToMember(ctx context.Context, members map[uint64]*membership.Member, m raftpb.Message) {
```

Comment:
Why pass `ctx` to this method if it immediately gets overwritten a few lines below?

I know this gets cleaned up in the other PR, but maybe it's better not to pass `ctx` to this method for now.